components/Dropdown/Dropdown.js

```diff
@@ -2,27 +2,29 @@ class Dropdown {
   constructor(element) {
     
     // Assign this.element to the dropdown element
-    this.element;
+    this.element = element;
     
     // Get the element with the ".dropdown-button" class found in the dropdown element (look at the HTML for context)
-    this.button = this.element.querySelector();
+    this.button = this.element.querySelector('.dropdown-button');
     
     // assign the reference to the ".dropdown-content" class found in the dropdown element
-    this.content;
+    this.content = document.querySelector('.dropdown-content');
     
     // Add a click handler to the button reference and call the toggleContent method.
-    this.button.addEventListener('click', () => {
-
-    })
+    this.button.addEventListener('click', () => this.toggleContent());
   }
 
   toggleContent() {
-    
     // Toggle the ".dropdown-hidden" class off and on
-    this.content;
-  }
+    this.content.classList.toggle('dropdown-hidden');
+  };
 }
 
 
 // Nothing to do here, just study what the code is doing and move on to the Dropdown class
-let dropdowns = document.querySelectorAll('.dropdown').forEach( dropdown => new Dropdown(dropdown));
\ No newline at end of file
+const dropdowns = document.querySelectorAll('.dropdown')
```

Comment:
I like how you separated the .forEach from the query selector so that you could console.log dropdowns.